domain/annotation/service/service.go

```diff
@@ -13,13 +13,14 @@ import (
 
 // State describes retrieval and persistence methods for annotations.
 type State interface {
-	// GetAnnotations retrieves all the annotations associated with a given ID. If no annotations
-	// are found, an empty map is returned.
+	// GetAnnotations retrieves all the annotations associated with a given ID.
+	// If no annotations are found, an empty map is returned.
 	GetAnnotations(ctx context.Context, ID annotations.ID) (map[string]string, error)
 
-	// SetAnnotations associates key/value annotation pairs with a given ID. If annotation already
-	// exists for the given ID, then it will be updated with the given value. First all annotations
-	// are deleted, then the given pairs are inserted, so unsetting an annotation is implicit.
+	// SetAnnotations associates key/value annotation pairs with a given ID.
+	// If annotation already exists for the given ID, then it will be updated
+	// with the given value. First all annotations are deleted, then the given
+	// pairs are inserted, so unsetting an annotation is implicit.
 	SetAnnotations(ctx context.Context, ID annotations.ID, annotations map[string]string) error
```

Comment:
This repeats content from the annotation/doc.go file. I'd remove it from there as it seems more relevant here.